.ci/opensource_jjb.yaml

```diff
@@ -43,10 +43,6 @@
             name: "do_package"
             default: true
             description: "Check tar, source and binary packages."
-        - bool:
-            name: "do_release"
-            default: false
-            description: "Publish package to release folder"
         - string:
```

Comment:
i guess we can remove this as well

Reply:
?

Reply:
release was removed from regular build and moved to the release pipeline